src/patterns.md

```diff
@@ -414,6 +389,7 @@ match tuple {
 >
 > _HalfOpenRangePattern_ :\
 > &nbsp;&nbsp; | _RangePatternBound_ `..`
+> &nbsp;&nbsp; | `..=` _RangePatternBound_
```

Comment:
Generally the grammar hasn't been using the leading-vert style (where the first alternation starts with `|`). I don't know why the original _HalfOpenRangePattern_ had it that way, it looks like an oversight. To stay consistent, could the first line change the `|` to `&nbsp;&nbsp;`?